### ISSUE TITLE
Make explanatory texts selectable and copyable

### ISSUE THREAD

**Issue by mk-pmb** [mumble-voip/mumble#4002]
**Context**
In which context would you expect your suggested feature to be useful? 

First-time setup of mumble for users who like to read text some other way than visual-only.

**Describe the feature you have in mind**

The explanatory texts in the wizard should allow text selection and copying, maybe using a read-only text input field.

**Describe alternatives you've considered**

* OCRing a screenshot. Not worth the effort for me.
* Trying to install a system-level program to coerce all text labels into text fields, or grab their text in other ways. Not worth the effort for me.

**Additional context**

Thanks for your work on this project!

**Comment by Krzmbrzl** [comment]
I assume you are referring to the audio wizard?

> First-time setup of mumble for users who like to read text some other way than visual-only. 

What exactly would such an application be concretely? Has this to do with screen readers?

**Comment by mk-pmb** [comment]
Yes, I'd like to copy the text to clipboard so I can send it over network to the computer that has my headphones connected, where it can be read by text-to-speech.

**Comment by Krzmbrzl** [comment]
I wanted to overhaul the wizard sometime soon anyways and while I do I'll try to make this happen.

Don't know if I get around doing this everywhere though... We'll see :)

**Comment by mk-pmb** [comment]
It just came to my mind that the accessibility of those texts was just a secondary problem. The primary problem I had was the wrong impression that I'd have to deal with them at that time. I wanted to use Mumble to set up rooms and permissions, but on launch it presented an audio settings wizard. My first impression was "oh no, I hope it hasn't screwed with any volume settings yet." Yeah, burned for life by a commercial "web phone" app some years ago.
Then I went fetch the sound card and microphone that I had in mind to use later, plugged it in, tried to play along as nicely as possible just to make sure the wizard won't go on a rampage trying to auto-"fix" missing information. There should have been a much earlier and very clear way to tell it "Don't bother with audio yet". I also didn't understand why I had to deal with volume levels anyway, because I already knew I'd want to use push-to-talk later, and another program will adjust volume levels.

**Comment by Krzmbrzl** [comment]
Uhm the wizard does have an abort button. So there is a way to skip over it.
It is enabled by default to avoid bug reports that simply occur because people didn't have the right settings.

**Comment by mk-pmb** [comment]
Indeed, and in an ideal world I could trust that abort to not break anything. Unfortunately I've seen software that misbehaves as badly as setting its best guess defaults if you abort their wizard. Also at that point I didn't know whether the wizard would be important and whether I could easily get it again later. An explicit "deal with audio later" button or similar could clarify that.

**Comment by Krzmbrzl** [comment]
Well I don't know. Just because other software messes up on this one shouldn't mean we have to clutter the UI with overly long texts. The problem I see is that `Deal with audio later` doesn't really fit on a normal-sized button.

**Comment by mk-pmb** [comment]
Indeed. When you have drafts for the new design, let me have a look and maybe I'll come up with an idea.

**Comment by Krzmbrzl** [comment]
For the accessibility this might be helpful:
* https://forum.qt.io/topic/15874/how-to-make-a-label-mouse-selectable/5
* https://doc.qt.io/qt-5/qlabel.html#textInteractionFlags-prop

**Comment by bendem** [comment]
> The problem I see is that `Deal with audio later` doesn't really fit on a normal-sized button.

I'd say if you wanted this, the first step of the wizard would have text explaining that you are going to setup audio, that's it's not mandatory but you should do it before trying to speak to others and that the wizard is always accessible and where if you skip it now. With that, you can have a button named "Skip" and it's pretty clear to everybody what it does.
